test/project_types/script/layers/application/create_script_test.rb

```diff
@@ -19,7 +19,9 @@
   let(:task_runner) { stub(compiled_type: compiled_type) }
   let(:project_creator) { stub }
   let(:project_directory) { '/path' }
-  let(:script_project) { TestHelpers::FakeScriptProject.new(language: language, directory: project_directory) }
+  let(:script_project) do
+    TestHelpers::FakeScriptProject.new(language: language, directory: project_directory, script_name: script_name)
```

Comment:
I feel I am missing something.  Why is this constructor taking in `directory`?

Reply:
Projects have a `directory` smart parameter